src/app/state/tabs/sql.rs

```diff
@@ -129,11 +55,11 @@ pub fn keyword_style() -> Style {
 pub struct SQLTabState<'app> {
     editor: TextArea<'app>,
     editor_editable: bool,
-    query: Option<Query>,
     query_results_state: Option<RefCell<TableState>>,
     result_batches: Option<Vec<RecordBatch>>,
     results_page: Option<usize>,
     execution_error: Option<ExecutionError>,
+    execution_task: Option<JoinHandle<Result<()>>>,
```

Comment:
Playing with the test code made me realize this would be useful to have for long running queries.